src/edu/csus/ecs/pc2/core/PacketHandler.java

```diff
@@ -1622,15 +1624,26 @@ private void confirmSubmission(Packet packet, ClientId fromId) {
 
         Clarification submittedClarification = (Clarification) PacketFactory.getObjectValue(packet, PacketFactory.CLARIFICATION);
         Clarification clarification = contest.acceptClarification(submittedClarification);
-
-        // Send to team
+           
+        IElementObject[] ultimateDestination = (IElementObject[]) PacketFactory.getObjectValue(packet, PacketFactory.DESTINATION);
+        // Send to judge rather than team
         Packet confirmPacket = PacketFactory.createClarSubmissionConfirm(contest.getClientId(), fromId, clarification);
         controller.sendToClient(confirmPacket);
 
         // Send to clients and other servers
         if (isServer()) {
             controller.sendToJudgesAndOthers(confirmPacket, true);
         }
+        //check if clarification is actually an announcement clarification.
+        if (clarification.getQuestion().equals("") && clarification.isAnswered()) {
```

Comment:
Maybe make a method in Clarification that tests if a clarification is an announcement by performing this test in the object?

Reply:
Will change.

Reply:
Fixed!